shared/domains/chelonia/internals.js

```diff
@@ -1167,7 +1176,7 @@ export default (sbp('sbp/selectors/register', {
               }).catch((e) => {
                 // Using console.error instead of logEvtError because this
                 // is a side-effect and not relevant for outgoing messages
-                console.error('Error stopping watching events after removing key', { contractID, foreignContract, foreignKeyName, fkUrl })
+                console.error('Error stopping watching events after removing key', { contractID, foreignContract, foreignKeyName, fkUrl }, e)
```

Comment:
Does this code get run on the server? If so, the `e` should be the first parameter. If not, it can be left as-is.

Reply:
This may or may not be run on the server

Reply:
Then we should probably move the `e` to the front